README.md

```diff
@@ -4,22 +4,96 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to a local or remote OpenShift cluster
+* have the operator installed on the cluster
+* have the plugin running
+* have the console running
+
```

Comment:
We should mention cert-manager operator here, that currently if you want tls config you need install cert-manager redhat version (community one will be supported later)